### PR TITLE
Create new Yoga home page

### PR DESCRIPTION
**Pull request by allyssonsantos**
Desktop:
image

Mobile: 
image

### CODE REVIEW

File: packages/doc/src/components/GlobalStyle/GlobalStyle.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    html, body {
+      color: #666;
+      font-family: 'neue-haas-grotesk-display';
+      letter-spacing: 0.5px;
+      height: 100%;
+      margin: 0;
+      padding: 0;
+      overflow: ${overflow ? 'hidden' : 'auto'};
```
Comment (victormath12) [inline packages/doc/src/components/GlobalStyle/GlobalStyle.jsx]: Should we follow the CSS Ordering here? like: 

```
- Layout Properties (position, float, clear, display)
- Box Model Properties (width, height, margin, padding)
- Visual Properties (color, background, border, box-shadow)
- Typography Properties (font-size, font-family, text-align, text-transform)
- Misc Properties (cursor, overflow, z-index)
```

File: packages/doc/src/components/GlobalStyle/GlobalStyle.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    h1, h2, h3, h4, h5, h6 {
+      color: #333;
+      font-weight: 300;
+      margin: 45px 0 20px;
```
Comment (victormath12) [inline packages/doc/src/components/GlobalStyle/GlobalStyle.jsx]: Should we follow the CSS Ordering here?

File: packages/doc/src/components/GlobalStyle/GlobalStyle.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    h1 {
+      font-size: 48px;
+      font-weight: 300;
+      margin: 0;
+
+      + p {
+        color: #6b6b78;
+        font-weight: 300;
+        margin: 10px 0 50px;
+      }
```
Comment (victormath12) [inline packages/doc/src/components/GlobalStyle/GlobalStyle.jsx]: Should we follow the CSS Ordering here?

File: packages/doc/src/components/Home/Box.jsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    margin-bottom: 0;
+  }
+
+  @media (max-width: 900px) {
+    flex-basis: 100%;
+    margin: 9px 0;
+
+    height: auto;
+  }
+`,
```
Comment (victormath12) [inline packages/doc/src/components/Home/Box.jsx]: I think that we can adjust this component following the CSS Ordering, what do you think?

File: packages/doc/src/components/Home/Title.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  font-weight: 300;
+  line-height: 64px;
+  margin: 0 0 24px;
+
+  @media (max-width: 900px) {
+    font-size: 20px;
+    line-height: 30px;
+    margin-bottom: 0;
+  }
+`;
```
Comment (victormath12) [inline packages/doc/src/components/Home/Title.jsx]: I think that we can adjust this component following the CSS Ordering, what do you think?

File: packages/doc/src/components/GlobalStyle/GlobalStyle.jsx (hunk 1 above)
Comment (allyssonsantos) [reply packages/doc/src/components/GlobalStyle/GlobalStyle.jsx]: Done!

File: packages/doc/src/components/GlobalStyle/GlobalStyle.jsx (hunk 3 above)
Comment (allyssonsantos) [reply packages/doc/src/components/GlobalStyle/GlobalStyle.jsx]: Done!

File: packages/doc/src/components/Home/Box.jsx (hunk 4 above)
Comment (allyssonsantos) [reply packages/doc/src/components/Home/Box.jsx]: Done!

File: packages/doc/src/components/Home/Title.jsx (hunk 5 above)
Comment (allyssonsantos) [reply packages/doc/src/components/Home/Title.jsx]: Done!

File: packages/doc/src/components/GlobalStyle/GlobalStyle.jsx (hunk 2 above)
Comment (allyssonsantos) [reply packages/doc/src/components/GlobalStyle/GlobalStyle.jsx]: Done!

**Pull request merged**